adagrams/game.py

```diff
@@ -1,13 +1,80 @@
 from random import randint
-
+letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
 def draw_letters():
-    pass
+    drawed_letters=[]
+    all_letters=[]
+    letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
+    for letter, quantity in letters_pool.items():
+        for i in range(quantity):
+            all_letters.append(letter)
+    while len(drawed_letters)<10:
+        random_number= randint(0,len(all_letters)-1) 
+        drawn_letter=all_letters[random_number]
+        drawed_letters.append(drawn_letter)
+        all_letters.pop(random_number)
+        letters_pool[drawn_letter] -= 1
```

Comment:
You get a random index and then use that random index to get a random letter from `all_letters`. Then you pop the random letter out of `all_letters`. Do you also need to decrement the value of the letter in `letters_pool`?

You only use `letters_pool` on lines 7-9 to build up the list `all_letters`. You don't use `letters_pool` for tracking, therefore we don't need to decrement the value of the letters in the dictionary because you're not doing anything with that information.